src/song-meta-data.h

```diff
@@ -25,28 +25,31 @@
 #ifndef _SONG_META_DATA_H
 #define _SONG_META_DATA_H
 
+#include <string>
+
 // An 'object' dealing with the meta data of a song.
-struct SongMetaData {
-  const char *title;
-  const char *artist;
-  const char *album;
-  const char *genre;
-  const char *composer;
-};
+// TODO Tucker Great opertunity for a class here
+typedef struct TrackMetadata {
```

Comment:
We don't need the typedef here, as `struct Foo` can be just used as type `Foo` in C++ directly.